bullet_server/Body[] body
bullet_server/SoftBody[] soft_body
bullet_server/Constraint[] constraint
# remove any existing bodies and constraint with above names
# instead of adding them
bool remove
---
bool success
